daprdocs/content/en/operations/support/support-release-policy.md

```diff
@@ -28,6 +28,17 @@ There will be at least 6 weeks between major.minor version releases giving users
 
 Patch support is for supported versions (current and previous).
 
+## Build variations
+
+The Dapr's sidecar image is published to GitHub Container Registry and Docker Registry. The default image contains all components. Since version 1.11, Dapr also offers a variation of the sidecar image, containing only stable components.
+
+* Default sidecar images: `daprio/daprd:<version>` or `ghcr.io/dapr/daprd:<version>`
+* Sidecar images for stable components: `daprio/daprd:<version>-stable` or `ghcr.io/dapr/daprd:<version>-stable`
+
+On Kubernetes, the sidecar image can be overwritten at the Deployment resource of the application via the `dapr.io/sidecar-image` annotation. See more about [Dapr's arguments and annotations]({{<ref "arguments-annotations-overview.md" >}})
```

Comment:
```suggestion
On Kubernetes, the sidecar image can be overwritten for the application Deployment resource with `dapr.io/sidecar-image` annotation. See more about [Dapr's arguments and annotations]({{<ref "arguments-annotations-overview.md" >}}). The default 'daprio/daprd:latest' image is used if not specified. 
```

Reply:
I do suggest that we also update https://docs.dapr.io/reference/arguments-annotations-overview/ into this PR for the dapr.io/sidecar-image annotations and have a link back to this topic. Like this:

"Dapr sidecar image. Default is daprio/daprd:latest. The Dapr sidecar will use this image instead of the latest default image. Use this when building your own custom image of Dapr and or [using the alternative stable Dapr image]({{<ref "support-release-policy.md/#build-variations" >}}) .

Reply:
Done.